### DISCUSSION TITLE
enable discussion categories

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#738] [General] [answered]
github discussions can have categories.  @sbwalker can request them on this thread: https://github.com/gh-community/discussions-feedback/discussions/153

![image](https://user-images.githubusercontent.com/311063/92761795-dda41600-f346-11ea-88c4-60a5dd2493a1.png)

**Comment by sbwalker** [comment, marked as answer]
Done

**Comment by kevind2000** [comment]
Any further guidance and instructions how to use the newly added "Categories"?
